pkg/util/kubeobjects/pod/pod.go

```diff
@@ -1,4 +1,4 @@
-package kubeobjects
```

Comment:
now you can simplify methods, from GetPod to `pod.Get`

Reply:
`pod.GetName` as well

Reply:
done in [542e38f](https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/542e38f34e74944a9b6e7e5db9623aa2d7969e7e)